ui/components/component-library/avatar-badge/avatar-badge.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box/box';
+import { BaseAvatar } from '../base-avatar';
+
+export const badgePosition = ['top', 'bottom'];
+
+export const AvatarBadge = ({
+  children,
+  badgeProps,
+  BadgeVariant,
+  ...props
+}) => {
+  return (
+    <Box className="avatar-badge">
+      <BaseAvatar className="avatar-badge__container" {...props}>
+        {/* Jazzicon , Avatar Account */}
+        {children}
+      </BaseAvatar>
```

Comment:
I think instead of using the `BaseAvatar` here we can put the responsibility on the engineer to choose which Avatar they would like to use. I understand that this also means that we are asking them to choose the size of the badge as well. 

```jsx
<AvatarWithBadge badge={<AvatarNetwork />}>
  <AvatarAccount />
</AvatarWithBadge >
```